src/main/utils/goUtils.ts

```diff
@@ -219,20 +236,26 @@ export class GoUtils {
                 return;
             }
 
-            // Files other than go.mod and *.go files are not necessary to build the dependency tree of used Go packages.
-            // Go files should be copied to allow running `go list -f "{{with .Module}}{{.Path}} {{.Version}}{{end}}" all`
-            // and to get the list package that are actually in use by the Go project.
-            if (curPath.endsWith('.go')) {
-                fs.copySync(curPath, destPath);
+            if (curPath.endsWith('_test.go')) {
                 return;
             }
```

Comment:
Why are you skipping the `_test.go` files? It will cause `go mod tidy` to skip the test dependencies.

Reply:
Good catch!, thanks!